### DISCUSSION TITLE
PlanetarySystemStacker not compatible with my new M1 MacBook Pro.

### DISCUSSION THREAD

**Discussion by jongreif** [Rolf-Hempel/PlanetarySystemStacker#55] [Q&A] [answered]
PSS not compatible with my new M1 MacBook Pro -- wants ARM64 compatibility.  Is there a fix?  I do have the latest version (0.8.22).  Thanks, Jon.

**Comment by jongreif** [comment, marked as answer]
> PSS not compatible with my new M1 MacBook Pro -- wants ARM64 compatibility. Is there a fix? I do have the latest version (0.8.22). Thanks, Jon.

Just discovered "Detailed Instructions for Installing PSS on an M1 Mac" in Cloudy Nights (https://www.cloudynights.com/topic/753290-planetarysystemstacker-installation-issues/page-2, post 44).  Will try that.  Thanks, Jon.

**Reply by Planet30** [reply]
Seems to be "broken" in Ventura. Ventura does not (seem to) allow to copy the Terminal app.

**Comment by Planet30** [comment]
Try this:

https://www.cloudynights.com/topic/753290-planetarysystemstacker-installation-issues/?p=12253036

`arch -x86_64 ....
`

**Comment by timing** [comment]
I've created a fork that does install on OSX, it also updates to Qt6. https://github.com/timing/PlanetarySystemStacker
